unison-src/transcripts/ability-order-doesnt-affect-hash.output.md

```diff
@@ -45,7 +45,5 @@ scratch/main> names term1
   Term
   Hash:   #8hum58rlih
   Names:  term1 term2
-  
-  Tip: Use `names.global` to see more results.
```

Comment:
I removed this tip because it's not going to be a reasonable course of action in most cases 🤷🏼‍♂️